<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Secret_Voldemort_Serveur</class>
 <widget class="QMainWindow" name="Secret_Voldemort_Serveur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server_CTR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>High Tower Text</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget{
	background-color:rgba(0,0,0,0);
}

QTabBar::tab {
	background: rgba(230, 230, 230, 20); 
	border: none; 
	padding: 15px;
	color:white;
} 

QTabBar::tab:hover { 
	background: rgba(230, 230, 230, 30); 
	margin-bottom: -1px; 
}

QTabBar::tab:selected { 
	background: rgba(230, 230, 230, 40); 
	margin-bottom: -1px; 
}

QTabWidget::pane { 
   border: none;
}

QTabWidget::tab_details{
	background: rgba(230, 230, 230, 20); 
}</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_details">
       <attribute name="title">
        <string>Détails de la partie</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_14">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>High Tower Text</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color:white;
}</string>
            </property>
            <property name="text">
             <string>Partie </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="input_gameStatus">
            <property name="font">
             <font>
              <family>High Tower Text</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color:white;
}</string>
            </property>
            <property name="text">
             <string>en cours</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>High Tower Text</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color:white;
}</string>
            </property>
            <property name="text">
             <string>Joueurs connectés : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="input_numberPlayerConnected">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color:white;
}</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="font">
           <font>
            <family>High Tower Text</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolBox::tab {
	background: rgba(250,250,250,20);
	border-left:1px solid white;
	color: white;
padding: 0px 0px 0px 5px;
}

QToolBox::tab:hover { 
	background: rgba(250,250,250,30);
}

QToolBox::tab:selected { 
	background: rgba(250,250,250,40);
}

QToolBox,
QToolBox &gt; *,
QToolBox &gt; QScrollArea &gt; #qt_scrollarea_viewport &gt; QWidget {
	background: transparent;
}



QToolBox QScrollBar:horizontal {
    border: none;
    background: rgba(250,250,250,40);
    height: 10px;
    margin: 0px 0px 0px 0px;
}
QToolBox QScrollBar::handle:horizontal {
    background: #FFFFFF;
    min-width: 10px;
}

QToolBox QScrollBar::add-line:horizontal {
    background: none;
    width: 0px;
	height: 0px;
}

QToolBox QScrollBar::sub-line:horizontal {
    background: none;
    width: 0px;
	height: 0px;
}

QToolBox QScrollBar:left-arrow:horizontal, QToolBox QScrollBar::right-arrow:horizontal {
    width: 0px;
    height: 0px;
    background: none;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_board">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>778</width>
             <height>321</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Plateau de jeu</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="layout_boardMangemort">
                <property name="spacing">
                 <number>0</number>
                </property>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="layout_boardPhenixOrder">
                <property name="spacing">
                 <number>0</number>
                </property>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_players">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Joueurs</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="layoutPlayers">
              <property name="spacing">
               <number>5</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_pile">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>83</width>
             <height>42</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Pile de loi</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="spacing">
             <number>5</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>System</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:white;
}</string>
              </property>
              <property name="text">
               <string>&gt;&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layoutPile">
              <property name="spacing">
               <number>5</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_log">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="log">
          <property name="font">
           <font>
            <family>High Tower Text</family>
            <pointsize>10</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit{
	background-color:rgba(250,250,250,5);
	color:white;
	border:none;
	border-left:1px solid white;
	padding:8px;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'High Tower Text'; font-size:10pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_maintenance">
       <attribute name="title">
        <string>Maintenance</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRestartTurn">
            <property name="font">
             <font>
              <family>High Tower Text</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color : rgba(250,250,250,20);
	color : #CCCCCC;
	padding-left : 20px;
	padding-right : 20px;
	padding-top : 5px;
	padding-bottom : 5px;
	margin:0px;
	margin-left:50px;
	margin-right:50px;
}

QPushButton::hover{
	background-color : rgba(250,250,250,40);
	color:white;
}</string>
            </property>
            <property name="text">
             <string>Revenir au début du tour</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonStartTurnWithInputs">
            <property name="font">
             <font>
              <family>High Tower Text</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color : rgba(250,250,250,20);
	color : #CCCCCC;
	padding-left : 20px;
	padding-right : 20px;
	padding-top : 5px;
	padding-bottom : 5px;
	margin-left:50px;
	margin-right:50px;
}

QPushButton::hover{
	background-color : rgba(250,250,250,40);
	color:white;
}</string>
            </property>
            <property name="text">
             <string>Lancer un début de tour avec les configurations de partie entrées en input</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRestartVote">
            <property name="font">
             <font>
              <family>High Tower Text</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color : rgba(250,250,250,20);
	color : #CCCCCC;
	padding-left : 20px;
	padding-right : 20px;
	padding-top : 5px;
	padding-bottom : 5px;
	margin-left:50px;
	margin-right:50px;
}

QPushButton::hover{
	background-color : rgba(250,250,250,40);
	color:white;
}</string>
            </property>
            <property name="text">
             <string>Recommencer le vote</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QCheckBox" name="buttonExpertMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>High Tower Text</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox{
	color:white;
}</string>
            </property>
            <property name="text">
             <string>Mode expert</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="version">
            <property name="styleSheet">
             <string notr="true">QLabel{
	color:white;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStart">
        <property name="font">
         <font>
          <family>High Tower Text</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color : #55FFFFFF;
	color : #DDDDDD;
	padding-left : 20px;
	padding-right : 20px;
	padding-top : 5px;
	padding-bottom : 5px;
	border:1px solid #CCCCCC;
}

QPushButton::hover{
	background-color : #DDFFFFFF;
	color : #101010;
	border:1px solid #808080;
}</string>
        </property>
        <property name="text">
         <string>Démarrer une partie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="buttonAntiSpoil">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>High Tower Text</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Mode caché</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
